crypto/enclave_encrypt.go

```diff
@@ -34,7 +38,9 @@ func LogHPKE() {
 	case hpke.KDF_HKDF_SHA256:
 		log.Debugf("\t*** HMAC-based key derivation function (HKDF) with SHA-256")
 	case hpke.KDF_HKDF_SHA384:
+		fallthrough
 	case hpke.KDF_HKDF_SHA512:
+		fallthrough
 	default:
 		log.Debugf("\t*** Unknown")
```

Comment:
Hpke doesn't provide a method to stringify the algorithm names 😓, so this approach is probably cleaner.